pom.xml

```diff
@@ -25,14 +25,7 @@
   <properties>
     <!-- The base directory for the maven release plugin to create the tag -->
     <tagBase>https://svn.jboss.org/repos/jbossas/projects/integration/tags/</tagBase>
-    <version.junit>[4.13.1,)</version.junit>
-    <version.narayana>6.0.0.Final</version.narayana>
-
-    <version.jnpserver>5.0.3.GA</version.jnpserver>
-    <version.org.jboss.logging.jboss-logging>3.5.0.Final</version.org.jboss.logging.jboss-logging>
-    <version.org.jboss.logging.jboss-logging-processor>2.2.1.Final</version.org.jboss.logging.jboss-logging-processor>
-    <version.jakarta.transaction.jakarta.transaction-api>2.0.1</version.jakarta.transaction.jakarta.transaction-api>
-    <version.jakarta.resource.jakarta.resource-api>2.0.0</version.jakarta.resource.jakarta.resource-api>
+    <version.narayana>7.0.0.Final</version.narayana>
```

Comment:
We need to break this circular dependency asap; the SPI is for integration and should stand independent from the implementations. I know that normally one would use a TCK for this but I'm pretty sure narayana is the only s/w that implements the SPI so a TCK would be overkill, unless someone has plenty of spare cycles to spend on it and can justify prioritising it above other work.

I did copy the test code over to the main narayana repo [1] a few years back, with a view to breaking the cycle, but until someone verifies that it provides the same test coverage we would need to keep it (I think it does but we need to check).

[1] https://github.com/jbosstm/narayana/blob/main/ArjunaJTA/spi/src/test/java/io/narayana/spi/SPIUnitTest.java
SPIUnitTest

Reply:
Running [tatletale](https://tattletale.jboss.org/) over wildfly code would show us which APIs it uses. I recall a few years back removing/updating WildFly in preparation for this (see issue JBTM-3722).